.github/workflows/deployBlocker.yml

```diff
@@ -13,28 +13,22 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Get URL, title, & number of new deploy blocker (issue)
-        if: ${{ github.event_name == 'issues' }}
-        env:
-          TITLE: ${{ github.event.issue.title }}
-        run: |
-          { echo "DEPLOY_BLOCKER_URL=${{ github.event.issue.html_url }}";
-          echo "DEPLOY_BLOCKER_NUMBER=${{ github.event.issue.number }}";
-          echo "DEPLOY_BLOCKER_TITLE=$(sed -e "s/'/'\\\\''/g; s/\`/\\\\\`/g; 1s/^/'/; \$s/\$/'/" <<< "$TITLE")";} >> "$GITHUB_ENV"
+      - uses: ./.github/actions/composite/setupGitForOSBotifyApp
+        id: setupGitForOSBotify
+        with:
+          GPG_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+          OS_BOTIFY_APP_ID: ${{ secrets.OS_BOTIFY_APP_ID }}
+          OS_BOTIFY_PRIVATE_KEY: ${{ secrets.OS_BOTIFY_PRIVATE_KEY }}
 
       - name: Update StagingDeployCash with new deploy blocker
-        uses: Expensify/App/.github/actions/javascript/createOrUpdateStagingDeploy@main
+        uses: ./.github/actions/javascript/createOrUpdateStagingDeploy
         with:
-          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          GITHUB_TOKEN: ${{ steps.setupGitForOSBotify.outputs.OS_BOTIFY_API_TOKEN }}
```

Comment:
Alternatively, I think we could just use `github.token` here. The GitHub Actions bot is also a GitHub App so has the higher rate limits. That's simpler, but then checklists would no longer be created by OSBotify. AFAIK that's not a problem though.

So I'm inclined to just use `github.token` here. Thoughts?

Reply:
Does `github.token` have the correct permissions to create? I thought it was usually just read only

Reply:
Well looking here: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

It looks like it should have read/write for everything it needs in the context of creating a checklist (issues, pull requests, actions)

Reply:
I'm happy with how code is now, but feel free to adjust if you prefer 👍